src/System.Runtime.Serialization.Json/tests/DataContractJsonSerializer.cs

```diff
@@ -2476,6 +2468,364 @@ public static void DCJS_RoundtrippingDateTime()
         }
     }
 
+    [Fact]
+    public static void DCJS_DateTimeFormatIsNull()
+    {
+        var settings = new DataContractJsonSerializerSettings();
+        Assert.Throws<ArgumentNullException>(() => settings.DateTimeFormat = new DateTimeFormat(null));
+        Assert.Throws<ArgumentNullException>(() => settings.DateTimeFormat = new DateTimeFormat("ddmmyyyyy", null));
+    }
+
+    [Fact]
+    public static void DCJS_NegativeDateTimeStylesTest_IncorrectFormat()
+    {
+        var dateTimeFormat = "yyyy-MM-ddTHH:mm:ss.fffK";
+        var original = new DateTime(2011, 1, 2, 3, 4, 5, 6);
+        var dcjsSettings = new DataContractJsonSerializerSettings()
+        {
+            DateTimeFormat = new DateTimeFormat(dateTimeFormat, CultureInfo.InvariantCulture)
+            {
+                DateTimeStyles = DateTimeStyles.None
+            },
+        };
+        var serializer = new DataContractJsonSerializer(typeof(DateTime), dcjsSettings);
+        var ms = new MemoryStream();
+        serializer.WriteObject(ms, original);
+        var serializedJsonValue = Encoding.UTF8.GetString(ms.ToArray());
+        serializedJsonValue = serializedJsonValue.Replace("2011", "         2011");
+        Assert.Throws<SerializationException>(() => DeserializeString<DateTime>(serializedJsonValue));
+    }
+
+    [Fact]
+    [ActiveIssue(18538, TestPlatforms.FreeBSD)]
+    [ActiveIssue(18538, TestPlatforms.Linux)]
+    [ActiveIssue(18538, TestPlatforms.NetBSD)]
+    [ActiveIssue(18539, TestPlatforms.Linux)]
```

Comment:
I wondered if it would work if we write the ActiveIssue like,

`[ActiveIssue(18538, TestPlatforms.FreeBSD | TestPlatforms.Linux | TestPlatforms.NetBSD)]`

It seems the whole test failed on two different issues. If that's the case, can you please split the test into two tests so that we have one test for one issue? Thanks!